frontend/lib/src/controllers/spending_breakdown_controller.dart

```diff
@@ -1,13 +1,16 @@
-import 'package:frontend/src/const/spending_breakdown_test_data.dart';
 import 'package:get/get.dart';
 
 import 'package:frontend/src/const/spending_breakdown_filter.dart';
+import 'package:frontend/src/models/spending_breakdown.dart';
+import 'package:frontend/src/services/account_service.dart';
 
 class SpendingBreakdownController extends GetxController {
   final Rx<BreakdownFilter> _selectedBreakdownFilter = breakdownFilters[0].obs;
   final RxDouble _totalSpent = 0.00.obs;
+  final RxList<SpendingBreakdown> _spendingList = <SpendingBreakdown>[].obs;
 
   BreakdownFilter get selectedBreakdownFilter => _selectedBreakdownFilter.value;
+  List<SpendingBreakdown> get spendingList => _spendingList;
```

Comment:
this is unnecessary, rxlist is already ok

Reply:
do i remove final in _spendingList?

Reply:
![image](https://github.com/framgia/sph-flutter/assets/114897466/f2e6016a-66eb-4609-a773-4dc8c54921fe)

Reply:
![image](https://github.com/framgia/sph-flutter/assets/95029803/bc07bc31-7d66-4148-bb3a-dfd546f22b10)

from your first commit

that was already good, was there any reason for the change?

Reply:
i wasn't sure if i should have it as a private attribute and just add getters. I'll revert it.